doc/source/conf.py

```diff
@@ -238,6 +238,7 @@ def __init__(self, version: str):
         r"http://localhost[:/].*",  # Ignore localhost links
         r"^http:/$",  # Ignore incomplete links
         # 403 Client Error: Forbidden for url.
+        "https://madewithml.com/#course",
```

Comment:
probably should drop the `#course`, and replace it with `/*` maybe.

could you add a commit on why it is forbidden?

you can test with curl by setting the user-agent header to mimic sphinx's behavior, if you think user-agent is the reason.